docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -605,8 +605,9 @@ How to **investigate**:
    1. Search in the logs around that time to find the log entry from when the compactor created the block ("compacted blocks" for log message)
    1. From the compactor log entry you found, pick the job ID from the `groupKey` field, f.ex. `0@9748515562602778029-merge--1645711200000-1645718400000`
    1. Then search the logs for the job ID and look for an entry with the message "compaction job finished" and `false` for the `success` field, this will show that the compactor failed uploading the block
+   1. If there is a "compaction job finished" with "success=`false`" and later a "compaction job finished" with "success=`true`", it means that the compaction was retried successfully. Note: this produces different block IDs so you won't necessarily see the skipped block being uploaded again.
```

Comment:
I think this wording is less ambiguous, to make sure you need to search for a retry of the same job:
```suggestion
   1. If you found a failed compaction job, as outlined in the previous step, try searching for a corresponding log message (for the same job ID) "compaction job finished" with "success=`true`". This will mean that the compaction job was retried successfully. Note: this should produce a different block ID from the failed upload.
```